libminifi/test/aws-tests/MockS3RequestSender.h

```diff
@@ -290,5 +290,5 @@ class MockS3RequestSender : public minifi::aws::s3::S3RequestSender {
   bool return_empty_result_ = false;
   bool is_listing_truncated_ = false;
   Aws::Auth::AWSCredentials credentials_;
-  Aws::Client::ClientConfiguration client_config_;
+  Aws::Client::ClientConfiguration* client_config_;
```

Comment:
should this be initialized to `nullptr`?

Reply:
Good point, updated in 1f4cadb5a3ddac9459b2d1a3a7520f21bb36f103